src/Index.js

```diff
@@ -493,7 +493,10 @@ Index.prototype.ttAdapter = function(params) {
       cb(content.hits);
     });
   };
-};
+},
+'ttAdapter is deprecated, there is no replacement planned, but open an issue on\n' +
```

Comment:
Message should say: ttAdapter is no more necessary, have a look at autocomplete.js (link)